app/src/androidTest/java/com/duckduckgo/app/privacymonitor/ui/PrivacyPracticesViewModelTest.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2017 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.privacymonitor.ui
+
+import android.arch.core.executor.testing.InstantTaskExecutorRule
+import android.arch.lifecycle.Observer
+import android.net.Uri
+import com.duckduckgo.app.privacymonitor.PrivacyMonitor
+import com.duckduckgo.app.privacymonitor.model.TermsOfService
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.whenever
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+
+
+class PrivacyPracticesViewModelTest {
+
+    @get:Rule
+    @Suppress("unused")
+    var instantTaskExecutorRule = InstantTaskExecutorRule()
+
+    private var viewStateObserver: Observer<PrivacyPracticesViewModel.ViewState> = mock()
+
+    private val testee: PrivacyPracticesViewModel by lazy {
+        val model = PrivacyPracticesViewModel()
+        model.viewState.observeForever(viewStateObserver)
+        model
```

Comment:
?

Reply:
Haha looks a bit weird right? It's a pattern I'm getting into for tests. The lazy load lets any all the mocking get setup before the model is generated. It's useful but if you find it really offensive I can refactor it out by generating the model in each test method

Reply:
It _looked_ weird, but i get it now. the stray `model` threw me as i was forgetting about implicit returns.

no action needed. 👍